feat: ja translate develop/api-reference/index

riywo opened the pull request:
Close #191

riywo committed:
feat: ja translate develop/api-reference/index

riywo commented:
@poppoerika Check this out!

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
 ### Flush cache
 
-Flushes all data from a cache
+キャッシュ内の全てのデータを真っ新にします。
```
Can we just say `フラッシュ`?

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
+| 名前      | 型   | 説明                      |
 | --------- | ------ | -------------------------------- |
-| cacheName | String | Name of the cache to be flushed. |
+| cacheName | String | 真っ新にするキャッシュの名前 |
```
same comment as https://github.com/momentohq/public-dev-docs/pull/196/files#r1186346023

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
-# API reference for Momento Cache
+# Momento Cache の API リファレンス
 
 ## Control APIs
```
`コントロールAPIs`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
 
 :::
 
 ## Data APIs
```
`データAPIs`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
 
 </details>
 
 ## Time to Live APIs
```
`生存期間(Time to Live)APIs`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
+| key       | String \| bytes | 値のTTL が減少されるキー |
+| ttl       | Duration | キャッシュ内の生存期間に設定したい秒数 |
 
 ### Collection data types (CDTs)
```
`コレクションデータ型(CDTs)`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
 ### Collection data types (CDTs)
 
-Collections may contain different types of structures depending on your use case. Supported data types are:
+集合型は、ユースケースに依っては異なる構造の型を含む場合もあります。サポートしているデータ型は以下の通りです:
```
`集合型` -> `コレクション型`

poppoerika commented on i18n/ja/docusaurus-plugin-content-docs/current/develop/api-reference/index.mdx:
```diff
@@ -12,257 +12,252 @@ import { SdkExampleSnippets } from "@site/src/components/SdkExampleSnippets";
-* [Lists](./list-collections.md) are a collection of ordered elements, sorted in the sequence each element was inserted.
-* [Sets](./set-collections.md) are an unordered collection of unique elements in string format.
-* [Sorted Sets](./sorted-set-collections.md) are an ordered collection of unique elements.  Each element contains a value:score pair.
+より詳しい利用方法の情報は、[集合データ型](./../datatypes.md)をご覧下さい。
```
https://github.com/momentohq/public-dev-docs/pull/196/files#r1186357403

riywo commented:
@poppoerika Addressed your comments! https://github.com/momentohq/public-dev-docs/compare/4a9792214d103319cda608d49b90cb63b484829c..6aa33c7236cf1aa9c7bcce882be4fda27ad46736
